src/targets/maven.ts

```diff
@@ -102,48 +93,115 @@ export class MavenTarget extends BaseTarget {
     );
   }
 
-  private getEnvVarOrDefault(envVar: string): string {
+  private getEnvVarOrDefault(envVar: OptionalConfig): string {
```

Comment:
Might be the time to use this: https://humanwhocodes.com/blog/2021/02/introducing-env-javascript-environment-variables/

Reply:
Did you have a look at this? Also, you can use `checkEnvForPrerequisite` from the `utils/env` module for required env variables.